integrations/deleting-an-integration.md

```diff
@@ -1,3 +0,0 @@
-# Deleting an Integration
```

Comment:
I deleted that mainly because I didn't find what else to add besides the guide:
<img width="810" alt="Screen Shot 2022-08-29 at 4 19 58 PM" src="https://user-images.githubusercontent.com/10411887/187315323-2e02eff0-4e25-4101-84d0-2f734d4f9fb7.png">

If we decide to keep it, we probably also want another page for `Edit Integration` with similar concerns, that the content will probably just repeat what's in the guide.

I do agree that having these pages will make the page structure, and they will be easier to find from the section menu at the left. So here I just link the page to the guide.

Reply:
Sorry for the back and forth, @likawind, but I made this comment before I looked at the other page you wrote (which is quite comprehensive). In light of that, let's actually go back to removing this page and the editing page.

Reply:
oh that sounds great!